src/data/Web Development/bugzilla.yaml

```diff
@@ -0,0 +1,7 @@
+name: Bugzilla
+description: |
+  ### Extra information for Bugzilla
```

Comment:
I don't think we need these `Extra information` headings.

Reply:
Yeah.